### PR TITLE
MNT move code from __init__ to rcca.py, fix Gaussian kernel, add requirements.txt, format code

### PR DESCRIPTION
**Pull request by robinthibaut**
Hello there,
I modified the code to solve the issue #16, and did some refactoring. I don't think the main code should be in an __init__ file, so I separated the two in "rcca.py" and "__init__.py". I specified an "__all__" in rcca.py, so that classes can be imported easily. I also added a requirement file.

### CODE REVIEW

**Review by TomDLT — APPROVED** [review]
Thanks for all this, although it would have been preferred having separate pull-requests for each change, both for reviewing and for future reference.

There is a conflict with recently merged #24, could you include this change here ?

Otherwise looks good to me.

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+numpy~=1.20.2
+h5py~=3.2.1
+scipy
+joblib
```
Comment (TomDLT) [inline requirements.txt:4]: Can you also add joblib to `setup.py` ?

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+numpy~=1.20.2
+h5py~=3.2.1
```
Comment (TomDLT) [inline requirements.txt:2]: How did you pick these version numbers?

Comment (robinthibaut) [reply requirements.txt]: They were generated by my IDE, they correspond to the versions I was using when making changes. I just replaced the "~=" by ">=" and filled the missing versions number.

**Comment by robinthibaut** [comment]
Hello, I fixed the remaining issues. 
In the future I'll submit PR one by one as suggested.
Kind regards,
Robin

File: requirements.txt (hunk 2 above)
Comment (TomDLT) [reply requirements.txt]: Well, if they are not the minimal version supported, I would rather have no version number. I will remove them.

**Comment by TomDLT** [comment]
Thanks !

**Pull request merged**